roles/smallfile/templates/smallfilejob.other_parameters.yaml.j2

```diff
@@ -15,16 +23,14 @@ files: {{ workload_args.files }}
 {% endif %}
 
 {% if workload_args.fsync is defined %}
-fsync: y
+fsync: {{ workload_args.fsync }}
```

Comment:
Given you're changing four different defaults, do we want to update the sample CRs `config/samples/smallfile/cr.yaml` (and/or in the docs) to add these parameters so we maintain  reasonable/functional defaults?

Reply:
so the full set of smallfile parameters is documented in smallfile github repo, don't have to repeat it.   This file defines a  subset of smallfile parameters that are usable with benchmark-operator.   The CI tests all the usable parameters now, I think.

Reply:
Right just doc the ones that we cover with the operator, I think the only two that are missing is the new `auto_pause` and `fsync`

Reply:
I also updated smallfile documentation to describe fsync and auto-pause and how to use them.